app/assets/javascripts/offendersummary/components/offenderDetails.js

```diff
@@ -98,49 +98,32 @@ const OffenderDetails = ({ offenderDetails }) => {
                                 <th>Mobile</th>
                                 <td className="qa-mobile">{ mobileNumber && mobileNumber.number || 'Unknown' }</td>
                             </tr>
+                            <tr>
+                                <th>Main address</th>
+                                <td>
+                                    { mainAddress && !mainAddress.noFixedAbode && (
+                                        <Fragment>
+                                            { mainAddress.buildingName && (<span className="qa-main-address-1">{ mainAddress.buildingName }<br/></span>) }
+                                            <span className="qa-main-address-2">{ mainAddress.addressNumber } { mainAddress.streetName }</span><br/>
```

Comment:
Does this look ok when there is no addressNumber of do you end up with a whitespace?